src/main/java/edu/hawaii/its/api/type/GroupingsAnnouncement.java

```diff
@@ -0,0 +1,67 @@
+package edu.hawaii.its.api.type;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+
+import java.time.LocalDateTime;
+
+public class GroupingsAnnouncement {
+    private String message = "";
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonFormat(pattern = "yyyyMMdd'T'HHmmss")
+    private LocalDateTime start;
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonFormat(pattern = "yyyyMMdd'T'HHmmss")
+    private LocalDateTime end;
+    public State state;
```

Comment:
Could you add a newline before all the @ JsonDeserialize annotations? Also I would move `public State state;` right under `private String message = "";`. One more thing, could you make the state a private variable? Thanks.